docs/standard-library/http/encoding.md

```diff
@@ -28,7 +28,7 @@ op upload(@body data: bytes): void; // Accept application/octet-stream
 
 **Default behavior:**
 
-- Encoded as `rfc7231` when used in a header
+- Encoded as `http-date` when used in a header
```

Comment:
Does this has any meaning outside of that openapi jsonschema format?

Reply:
The meaning comes from RFC 9110, the current HTTP RFC (obsoletes RFC 7231), specifically

https://datatracker.ietf.org/doc/html/rfc9110#name-date-time-formats

Reply:
Hhm then maybe we should consider renaming the known encoding and deprecating the old one.

Reply:
RFC7231 seems more general. PHP at least uses this constant to refer to that, C# uses 1123... I think we should use a general name for it unless http-date is a unique thing.